src/v0/destinations/braze/deleteUsers.js

```diff
@@ -21,11 +21,12 @@ const userDeletionHandler = async (userAttributes, config) => {
   }
   // Endpoints different for different data centers.
   // DOC: https://www.braze.com/docs/user_guide/administrative/access_braze/braze_instances/
+  // Example Data Center: "EU-01", "US-01"
   let endPoint;
   const endpointPath = '/users/delete';
   const dataCenterArr = dataCenter.trim().split('-');
   if (dataCenterArr[0].toLowerCase() === 'eu') {
-    endPoint = 'https://rest.fra-01.braze.eu/users/delete';
+    endPoint = `https://rest.fra-${dataCenterArr[1]}.braze.eu/users/delete`;
```

Comment:
For both the if and else part it is `dataCenterArr[1]` or for one it is index 0 and for another it is 1.

Reply:
actual endpoint id different @ItsSudip rest.iad for us and rest.fra for eu

Reply:
Got it